jobs/views.py

```diff
@@ -100,25 +102,79 @@ def confirm_submission(request):
     )
 
 
-class JobCreate(SuccessMessageMixin, CreateView):
-    model = Job
-    template_name = 'jobs/job_edit.html'
-    form_class = JobForm
-    success_url = reverse_lazy('jobs:confirm_submission')
+def create_job(request):
+    job_form = JobForm()
     success_message = 'Your job offer was added to our database, \
                     you will receive further information shortly.'
-
-    def form_valid(self, form):
-        return super(JobCreate, self).form_valid(form)
+    if request.method == 'POST':
```

Comment:
this could easily have stayed within the form_valid method of the CreateView and saved some code - but I know there is disagreement on use of CBVs so fine like this if you prefer it